src/databricks/labs/ucx/hive_metastore/catalog_schema.py

```diff
@@ -27,14 +29,31 @@ def __init__(
         self._principal_grants = principal_grants
         self._backend = sql_backend
 
+    def create_ucx_catalog(self, prompts: Prompts, *, properties: dict[str, str] | None = None) -> None:
+        """Create the UCX catalog.
+
+        Args:
+            prompts : Prompts
+                The prompts object to use for interactive input.
+            properties : (dict[str, str] | None), default None
+                The properties to pass to the catalog. If None, no properties are passed.
+        """
+        try:
+            self._create_catalog_validate(self.UCX_CATALOG, prompts, properties=properties)
```

Comment:
make it part of the `WorkspaceConfig`, otherwise we can't do integration testing for it. this will also make it impossible to test it internally by multiple people on the same workspace/account (we have this case and people will ask for it if it's missing).

also there might be some  customer need to use a different name because they have different naming strategy than this